### ISSUE TITLE
Module name values need to be prefixed with `Elixir.` in configuration files

### ISSUE THREAD

**Issue by flupke** [bitwalker/conform#35]
For example here is the schema generated for a conf entry that takes a module name:

```
    "backend": [
      to: "my_app.Elixir.MyApp.backend",
      datatype: :atom,
      default: MyApp.SomeBackend
    ],
```

In the conf file, you have to prefix the value with `Elixir.`:

```
backend = Elixir.MyApp.SomeBackend
```

**Comment by bitwalker** [comment]
I can address this I think without overcomplicating things, I'll take a look when my current set of changes is ready to be released.

**Comment by bitwalker** [comment]
This is now fixed in master (staged for 2.4.0 release)
